karspexet/templates/home.html

```diff
@@ -1,39 +1,43 @@
 {% extends "base.html" %}
 {% load cms_tags  sekizai_tags fontawesome %}
 {% block content %}
```

Comment:
{% clock content %} 
...
{% endblock %}
is a block and thus the indention might be good :)